crates/router/src/services.rs

```diff
@@ -29,7 +28,6 @@ impl Store {
     pub async fn new(config: &crate::configs::settings::Settings, test_transaction: bool) -> Self {
         Self {
             master_pool: diesel_make_pg_pool(&config.master_database, test_transaction).await,
-            #[cfg(feature = "olap")]
```

Comment:
we need to create feature flag to create connections, oltp cases the connections need not to be established to the databases.